rpm_spec/manageiq.spec.in

```diff
@@ -87,6 +87,10 @@ cd %{_builddir}
 %{__cp} -r %{gemset_builddir}/vmdb %{buildroot}%{gemset_root}
 install -m644 %{gemset_builddir}/enable %{buildroot}%{gemset_root}
 
+#copy systemd unit files
+%{__cp} %{appliance_builddir}/build/systemd_units/* %{buildroot}/etc/systemd/system
```

Comment:
I assume we're going to move the unit files in manageiq-appliance repo to this new location (build/systemd_units)?

Reply:
Or maybe you meant `%{core_builddir}` (for files you're adding in the core PR https://github.com/ManageIQ/manageiq/pull/20983)?

Reply:
:+1: yes these are intended to come from core, thanks I'm not 100% on what all of these macros relate to

Reply:
By default, files that come from the core repo is included in the `manageiq-core` rpm, and files that come from plugins are included in `manageiq-gemset` rpm.

The `%files` sections determine which files should be included in which rpm. When files are copied from 2 different places into a single location (in this case, unit files from core repo and plugin repos both going into the same systemd directory), `%files` section of each rpm should only list files that belong to that rpm.

You can list them all one by one (which is too much for this case, as there are many files!), or it's easy if each set starts with a different prefix or contain specific word(s) (for example, manageiq-core-xxx vs manageiq-plugin-xxx), but it doesn't look like that's the case here?  Another option is to put all unit files into a rpm, rather than going by where they came from). All rpms must be installed together anyway, so we won't have a case where some files will be missing.

Either way, I think it's a good idea to name files in a way it can be identified as manageiq owned when files are going into a system owned directory.

Reply:
We can certainly add a `manageiq-` prefix to the unit names, we'll have to do something about the previous files though.  Is there a way for us to remove old files on install?

I'm not sure about actually naming the services `manageiq-core-*` or `manageiq-plugin-*` because it doesn't matter at all after the "which rpm do these go in" step and it'll show up in `systemctl status` with those long names.  There aren't too many core units so maybe we could enumerate those by name and everything else goes in the gemset?

Reply:
Yes, there are ways to remove old files. You can use different scriptlets, depending on when you want to remove the files: https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/

As for the names, I wasn't suggesting to use `managei-core-` or `manageiq-plugins-`, those were just examples to illustrate how naming can help %files sections 😄

Reply:
> I assume we're going to move the unit files in manageiq-appliance repo to this new location (build/systemd_units)?

Note: as long as we're cleaning up old files I think it'd be better to put these in `/lib/systemd/system` which is where the current `evmserverd.service` is, so I've updated where we're copying these to match that

Reply:
@simaishi okay I've enumerated the core workers in the manageiq-core rpm_spec and the provider workers in the manageiq-gemset rpm_spec